src/module-elasticsuite-catalog/etc/adminhtml/system.xml

```diff
@@ -41,6 +41,7 @@
                     <comment><![CDATA[Maximum number of categories to display in autocomplete results.]]></comment>
                 </field>
             </group>
+
```

Comment:
So the idea is to remove that "empty" change on that file.

Reply:
I removed this line break. Regards